wazuh/wazuh-dashboard-plugins
# Clear session button text is displayed in full

### Description

The clear session button in the ruleset test view now displays the complete text. 

### Issues Resolved

- #6870 

### Evidence

![image](https://github.com/user-attachments/assets/3190d00e-51c1-4275-a81b-0eaf41ae955d)


### Test

1. Navigate to 'Ruleset test'
2. See the button

### Check List
